homeassistant/components/drop_connect/sensor.py

```diff
@@ -0,0 +1,272 @@
+    "high_system_pressure": DROPSensorEntityDescription(
+        key="high_system_pressure",
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement="psi",
+        suggested_display_precision=0,
+        value_fn=lambda device: device.high_system_pressure,
+    ),
+    "low_system_pressure": DROPSensorEntityDescription(
+        key="low_system_pressure",
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement="psi",
+        suggested_display_precision=0,
+        value_fn=lambda device: device.low_system_pressure,
+    ),
+    "battery": DROPSensorEntityDescription(
+        key="battery",
+        device_class=SensorDeviceClass.BATTERY,
+        native_unit_of_measurement="%",
+        suggested_display_precision=0,
+        value_fn=lambda device: device.battery,
+    ),
+    "temperature": DROPSensorEntityDescription(
+        key="temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement="°F",
+        suggested_display_precision=1,
+        value_fn=lambda device: device.temperature,
+    ),
+    "inlet_tds": DROPSensorEntityDescription(
+        key="inlet_tds",
+        icon=TDS_ICON,
+        native_unit_of_measurement="ppm",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.inlet_tds,
+    ),
+    "outlet_tds": DROPSensorEntityDescription(
+        key="outlet_tds",
+        icon=TDS_ICON,
+        native_unit_of_measurement="ppm",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.outlet_tds,
+    ),
+    "cart1": DROPSensorEntityDescription(
+        key="cart1",
+        icon=GAUGE_ICON,
+        native_unit_of_measurement="%",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.cart1,
+    ),
+    "cart2": DROPSensorEntityDescription(
+        key="cart2",
+        icon=GAUGE_ICON,
+        native_unit_of_measurement="%",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.cart2,
+    ),
+    "cart3": DROPSensorEntityDescription(
+        key="cart3",
+        icon=GAUGE_ICON,
+        native_unit_of_measurement="%",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.cart3,
+    ),
+}
+
+
+class DROPSensor(DROPEntity, SensorEntity):
+    """Representation of a DROP sensor."""
+
+    entity_description: DROPSensorEntityDescription
+
+    def __init__(self, device, entity_description) -> None:
+        """Initialize the sensor."""
+        super().__init__(entity_description.key, device)
+        self.entity_description = entity_description
+        self.device = device
+        self._attr_translation_key = entity_description.key
```

Comment:
Yes and no. We now also have a thing called device class translations. So if you have a sensor entity with a device class you can remove the translation key and it will default to the default name of that device class. So for a temperature sensor you can remove the translation key and it will always translate it to temperature.

Reply:
Since you have more entities with the same device class, I would actually suggest to add a field `translation_key=""` to every description and omit it at the places where you use device class naming

Reply:
I have no reason for my SensorEntityDescription `key` value to be different than the `translation_key` value, so I simplified all the entity descriptions with that one line of code.  I can add the translation keys to the entity descriptions if that approach isn't permissible.